src/decompose_mesh/module_database.f90

```diff
@@ -0,0 +1,457 @@
+module module_database
+  
+  use shared_parameters, only: NGNOD, NGNOD2D, LOCAL_PATH, MSL=>MAX_STRING_LEN
+ 
+  integer                                     :: nE_loc
+  integer, dimension(:),  allocatable         :: loc2glob_elmnt
+  integer, dimension(:),  allocatable         :: glob2loc_elmnt
+  
+  integer                                     :: nnodes_loc
+  integer, dimension(:),  allocatable         :: loc2glob_nodes 
+  integer, dimension(:),  allocatable         :: glob2loc_nodes
+
+  integer                                     :: size_adjancy
+  integer, dimension(:),  allocatable         :: adjcy, id_adjcy
+
+  integer                                     :: max_elmnts_by_node
+  integer, dimension(:),  allocatable         :: nelmnts_by_node
+  integer, dimension(:,:),allocatable         :: elmnts_by_node
+
+  integer, private, parameter                 :: IIN_database=49
+  integer, private                            :: ier
+  integer, private, dimension(:), allocatable :: node_loc
+
+contains
+
+!----------------------------------
+!
+!  set upt the arrays for database
```

Comment:
Fixed.

On 02/10/2016 07:28 PM, Elliott Sales de Andrade wrote:
> In src/decompose_mesh/module_database.f90
> <https://github.com/geodynamics/specfem3d/pull/694#discussion_r52501241>:
>
>> +  integer                                     :: size_adjancy
>> +  integer, dimension(:),  allocatable         :: adjcy, id_adjcy
>> +
>> +  integer                                     :: max_elmnts_by_node
>> +  integer, dimension(:),  allocatable         :: nelmnts_by_node
>> +  integer, dimension(:,:),allocatable         :: elmnts_by_node
>> +
>> +  integer, private, parameter                 :: IIN_database=49
>> +  integer, private                            :: ier
>> +  integer, private, dimension(:), allocatable :: node_loc
>> +
>> +contains
>> +
>> +!----------------------------------
>> +!
>> +!  set upt the arrays for database
>
> upt -> up
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geodynamics/specfem3d/pull/694/files#r52501241>.
>

-- 
Dimitri Komatitsch
CNRS Research Director (DR CNRS), Laboratory of Mechanics and Acoustics,
UPR 7051, Marseille, France    http://komatitsch.free.fr